doc/rados/operations/pools.rst

```diff
@@ -653,227 +644,234 @@ You may set values for the following keys:
 
 :Type: Double
 
 
 ``cache_target_dirty_high_ratio``
 
-:Description: see cache_target_dirty_high_ratio_
+:Description: See cache_target_dirty_high_ratio_.
 
 :Type: Double
 
 
 ``cache_target_full_ratio``
 
-:Description: see cache_target_full_ratio_
+:Description: See cache_target_full_ratio_.
 
 :Type: Double
 
 
 ``target_max_bytes``
 
-:Description: see target_max_bytes_
+:Description: See target_max_bytes_.
 
 :Type: Integer
 
 
 ``target_max_objects``
 
-:Description: see target_max_objects_
+:Description: See target_max_objects_.
 
 :Type: Integer
 
 
 ``cache_min_flush_age``
 
-:Description: see cache_min_flush_age_
+:Description: See cache_min_flush_age_.
 
 :Type: Integer
 
 
 ``cache_min_evict_age``
 
-:Description: see cache_min_evict_age_
+:Description: See cache_min_evict_age_.
 
 :Type: Integer
 
 
 ``fast_read``
 
-:Description: see fast_read_
+:Description: See fast_read_.
 
 :Type: Boolean
 
 
 ``scrub_min_interval``
 
-:Description: see scrub_min_interval_
+:Description: See scrub_min_interval_.
 
 :Type: Double
 
 
 ``scrub_max_interval``
 
-:Description: see scrub_max_interval_
+:Description: See scrub_max_interval_.
 
 :Type: Double
 
 
 ``deep_scrub_interval``
 
-:Description: see deep_scrub_interval_
+:Description: See deep_scrub_interval_.
 
 :Type: Double
 
 
 ``allow_ec_overwrites``
 
-:Description: see allow_ec_overwrites_
+:Description: See allow_ec_overwrites_.
 
 :Type: Boolean
 
 
 ``recovery_priority``
 
-:Description: see recovery_priority_
+:Description: See recovery_priority_.
 
 :Type: Integer
 
 
 ``recovery_op_priority``
 
-:Description: see recovery_op_priority_
+:Description: See recovery_op_priority_.
 
 :Type: Integer
 
 
 Setting the Number of RADOS Object Replicas
 ===========================================
 
-To set the number of object replicas on a replicated pool, execute the following:
+To set the number of object replicas on a replicated pool, run the following
+command:
 
 .. prompt:: bash $
 
    ceph osd pool set {poolname} size {num-replicas}
 
-.. important:: The ``{num-replicas}`` includes the object itself.
-   If you want the object and two copies of the object for a total of
-   three instances of the object, specify ``3``.
-
-For example:
+.. important:: The ``{num-replicas}`` argument includes the object itself.  For
+   example, if you want there to be two replicas of the object in addition to
+   the original object (for a total of three instances of the object) specify
+   ``3`` by running the following command:
 
 .. prompt:: bash $
 
    ceph osd pool set data size 3
 
-You may execute this command for each pool. **Note:** An object might accept
-I/Os in degraded mode with fewer than ``pool size`` replicas.  To set a minimum
-number of required replicas for I/O, you should use the ``min_size`` setting.
-For example:
+You may run the above command for each pool. 
+
+.. Note:: An object might accept I/Os in degraded mode with fewer than ``pool
```

Comment:
@zdover23 - Should _degraded_ be enclosed in double backticks?

Reply:
I don't think so in this caase